libyul/optimiser/Suite.cpp

```diff
@@ -260,7 +260,7 @@ map<string, char> const& OptimiserSuite::stepNameToAbbreviationMap()
 		{LoadResolver::name,                  'L'},
 		{LoopInvariantCodeMotion::name,       'M'},
 		{ReasoningBasedSimplifier::name,      'R'},
-		{RedundantAssignEliminator::name,     'r'},
+		{UnusedAssignEliminator::name,        'r'},
```

Comment:
I would still keep this at `r`

Reply:
Yeah, I think it's better to keep it. Change would require finding it when used in step sequences which is error-prone. Also, `u` is already taken and `U` will probably be taken by the other eliminator...